packages/office-addin-dev-settings/src/sideload.ts

```diff
@@ -237,36 +237,34 @@ async function getOutlookVersion(): Promise<string | undefined> {
   }
 }
 
-async function getOutlookExePath(): Promise<string> {
+async function getWXPOExePath(app: OfficeApp): Promise<string> {
+  let HostApp: string = "";
```

Comment:
JavaScript convention is lowercase first letter variable names.

Reply:
Got it. I will change it. Thanks!